Background.md

```diff
@@ -0,0 +1,47 @@
+# SOLID Quest - Achtergrondinformatie
+
+Dit bestand is een verzameling van de kennis en terminologie die we hebben vergaard tijdens onze quest:
+- [Data pods vs data spaces](https://github.com/kadaster-labs/solid-quest/blob/Documentatie/Background.md#data-pods-vs-data-spaces)
```

Comment:
Ah ... de links hoeven alleen het laatste stukje te zijn (vanaf de `#` - dat betekent 'in deze pagina, spring naar iets met het id van <wat achter de `#` staat>)